Frontend/ui-common/src/Engineering/Helpers/FileProvider/Utils.js

```diff
@@ -57,4 +57,14 @@ export const loadFileInDUI = function (fileName) {
   } else {
     return "";
   }
+}
+
+export const stringify = function (obj) {
```

Comment:
Can we rename it to stringifyJS as it's a predefined function name as well. It shouldn't coincide with any other function in future.

Reply:
ok